e2e-tests/native/test/specs/navigateToDashboard.e2e.ts

```diff
@@ -0,0 +1,31 @@
+import LandingPage from '../pageobjects/landing.page'
+import DashboardPage from '../pageobjects/dashboard.page'
+import {hideKeyboard, swipe, SwipeDirection} from "../Action";
+
+describe('navigate to dashboard', () => {
+
+  it('filter and navigate to City', async () => {
+    const filter = 'wirschaffendas'
+
+    expect(await LandingPage.exists()).toBeTruthy()
+
+    const cities = await LandingPage.cities
+    const search = await LandingPage.search
+
+    expect(cities.length).toBeGreaterThan(0)
+    await search.click()
+    await search.addValue(filter)
+
+    await hideKeyboard()
+    await swipe(SwipeDirection.Down)
```

Comment:
Do we still need both? Why?

Reply:
Because, the swiping does not work to hide the keyboard on iOS. At least I could not make it work. On Android you can actually say `driver.hideKeyboard('swipeDown')`, but iOS devices don't support this.

Reply:
Okay, not sure I understand but I guess you know what you do here :D